src/components/Pagination.jsx

```diff
@@ -0,0 +1,56 @@
+import PropTypes from "prop-types";
+import { GrFormNext, GrFormPrevious } from "react-icons/gr";
+
+const Pagination = ({ setCurrentPage, currentPage }) => {
+  const numberOfPages = 15;
+  const pages = Array.from({ length: numberOfPages }, (_, index) => index + 1);
+
+  return (
+    <div className="d-flex justify-content-center">
+      {currentPage > 1 && (
+        <button
+          className="prev-btn btn btn-warning fw-bolder p-1 rounded-circle mx-1"
+          onClick={() => setCurrentPage(currentPage - 1)}
+        >
+          <GrFormPrevious />
+        </button>
+      )}
+
+      {pages
+        .slice(
+          Math.max(0, currentPage - 3),
```

Comment:
3 seems like the magic number to me. Please avoid this practice. store these values in the variable just like `numberOfPages`.